app/views/SecurityCamerasChangeView.scala.html

```diff
@@ -44,7 +44,7 @@ <h1 class="govuk-heading-l">@messages("securityCamerasChange.heading")</h1>
         @govukInput(
             InputViewModel(
                 field = form("value"),
-                label = Label()
+                label = Label(classes = "govuk-label--s govuk-visually-hidden", content = Text(messages("securityCamerasChange.title")))
```

Comment:
```suggestion
label = LabelViewModel(messages("securityCamerasChange.title")).asPageHeading(LabelSize.Large)
  
``` we can remove the H1 code